src/components/P5ChallengeTaskStepper/P5ChallengeTaskStepper.scss

```diff
@@ -0,0 +1,182 @@
+.task-stepper-container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+  padding: 40px;
+
+  @media (max-width: 768px) {
+    gap: 12px;
+  }
+
+  @media (max-width: 480px) {
+    gap: 8px;
+  }
+}
+
+.step-button {
+  padding: 2px 16px;
+  background-color: #dadada;
+  border-radius: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+  width: fit-content;
+  text-align: center;
+  font-size: 12px;
+  font-family: "Corben", sans-serif;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.36px;
+  word-wrap: break-word;
+
+  &.completed {
+    background-color: green; /* Green when step is completed */
+    color: white;
+  }
+  &.current{
+    background-color:#0954B0; 
+    color: white;
+  }
+  &.inactive{
+    background-color: #dadada;
+    color:black;
+  }
+//   .button-text {
+//     text-align: center;
+//     color: black;
+//     font-size: 12px;
+//     font-family: "Corben", sans-serif;
+//     font-weight: 700;
+//     line-height: 24px;
+//     letter-spacing: 0.36px;
+//     word-wrap: break-word;
+//   }
+
+  @media (max-width: 768px) {
+    padding: 6px 14px;
+    .button-text {
+      font-size: 12px;
+    }
+  }
+
+  @media (max-width: 480px) {
+    padding: 4px 12px;
+    .button-text {
+      font-size: 10px;
+    }
+  }
+}
+
+.task-details {
+  width: 80%;
```

Comment:
The width should be 100% to more closely align with the figma design

Reply:
Resolved